_ols-1/week15.md

```diff
@@ -12,18 +12,18 @@ title: Week 15 - Final presentations & congratulations!
 # This week
 
 During this week's cohort call, we will:
-- 
+- Witness the final presentations from the project leads registered in Group 1 and see them graduate.
 
 # Resources
 
 - [Notes](https://docs.google.com/document/d/1zfn5hLldbti-K4L244_4A6wGc80FymKTeixN8cWXq2Q/edit?usp=sharing)
 - Recording (available on the [YouTube channel](https://www.youtube.com/channel/UCs12-ZgnDJOWIWN3Vo1XHXA/) after the call)
 
-
 # Before this meeting
 
-- 
+- Come prepared to share a 5 mins presentation of your project 
```

Comment:
3 minutes or 5 - rehearsals notes said 5 minutes

Reply:
It originally said 2 minutes here. Should we go for the full 5 minutes?